client/tests/qunit/tests/workflow_editor_tests.js

```diff
@@ -1280,13 +1288,9 @@ QUnit.test("simple mapping over collection outputs works correctly", function (a
     this.verifyNotAttachable(assert, testTerminal1, connectedOutput);
 });
 
-QUnit.test("node mapping state over collection outputs works correctly", function (assert) {
-    const inputTerminal1 = this.newInputTerminal();
-    const outputCollectionTerminal1 = this.newOutputCollectionTerminal("list");
-    assert.ok(!inputTerminal1.node.mapOver);
-    const connector = new Connector({}, outputCollectionTerminal1, inputTerminal1);
-    outputCollectionTerminal1.connect(connector);
-    assert.ok(inputTerminal1.node.mapOver);
-    inputTerminal1.disconnect(connector);
-    assert.ok(!inputTerminal1.node.mapOver);
-});
\ No newline at end of file
+QUnit.test("node input parameter mapping state over collection outputs works correctly", function (assert) {
+    const inputTerminal = this.newInputTerminal();
+    this.verifyDefaultMapOver(assert, inputTerminal);
+    const inputParameterTerminal = this.newInputParameterTerminal();
+    this.verifyDefaultMapOver(assert, inputParameterTerminal);
```

Comment:
Can you break these up, these are 2 distinct tests, right ?

Reply:
I think its fine to leave them together since both test if the default/basic mapping of collections to non-collection inputs is properly handled (for data and parameter inputs).

Reply:
Ok, I did split them.